crates/sncast/src/helpers/fee.rs

```diff
@@ -225,3 +233,29 @@ macro_rules! impl_payable_transaction {
         }
     };
 }
+
+impl FromStr for FeeToken {
+    type Err = String;
+
+    fn from_str(s: &str) -> Result<Self, Self::Err> {
+        match s.to_lowercase().as_str() {
+            "eth" => Ok(FeeToken::Eth),
+            "strk" => Ok(FeeToken::Strk),
+            _ => Err(String::from("Possible values: eth, strk")),
+        }
+    }
+}
```

Comment:
Why do we need to implement this if the function below is using an already parsed `FeeToken`?

Reply:
I've updated the logic, It seems to me that FromStr create a clear parsing interface